3 - PB/Documentazione interna/Verbali/24-02-29/24-02-29.typ

```diff
@@ -0,0 +1,36 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "29/02/24",
+  subTitle: "Meeting post colloquio con Proponente",
+  docType: "verbale",
+  authors: (
+    "Banzato Alessio",
+  ),
+  timeStart: "17:10",
+  timeEnd: "17:25",
+);
+
+= Ordine del giorno
+
+A seguito dell'incontro con il Proponente, il gruppo ha svolto un meeting interno riguardante:
+
+- Considerazioni scaturite dal meeting esterno;
+- Pianificazione.
+
+== Considerazioni scaturite dal meeting esterno
+
+Il gruppo:
+
+- ha esaminato le osservazioni del Proponente riguardo lo schema ER e ha concordato sul considerarlo come definitivo;
+- si è ritenuto soddisfatto del mock-up presentato e della discussione avuta con il Proponente, che ha portato a una rivalutazione di alcune funzionalità;
+- ha esaminato le considerazioni fatte riguardanti architettura e design, e ritiene essenziale anche un riscontro da parte del #cardin, al quale sono stati esposti i dubbi via mail.
+
+== To do
+
+Il gruppo ha fatto il punto sulla situazione dello sprint in corso, segnalando un andamento positivo e rimanendo quindi fiducioso sull'andamento dei lavori.
```

Comment:
Il gruppo ha fatto il punto sulla situazione dello sprint in corso, segnalando un andamento positivo sull'andamento dei lavori.